GP05MOAS-GL469/GP05MOAS-GL469_R00002_ingest.csv

```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL469/R00002/merged/gp_*.mrg,GP05MOAS-GL469-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL469/R00002/merged/gp_*.mrg,GP05MOAS-GL469-01-FLORDM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL469/R00002/merged/gp_*.mrg,GP05MOAS-GL469-02-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL469/R00002/merged/gp_*.mrg,GP05MOAS-GL469-04-CTDGVM000,recovered_host,Available,
```

Comment:
This looks like a telemetered ingest file, but the filename is R00002.

Reply:
Yeah I see now that the driver/parers are telemetered parsers. File mask and data_source are correct for a recovered ingestion. Will push updates to driver/parser names.